Libplanet/Net/Swarm.cs

```diff
@@ -737,20 +737,30 @@ private async Task BindingProxies()
         {
             while (Running)
             {
-                NetworkStream stream =
-                    await _turnClient.AcceptRelayedStreamAsync();
-
-                #pragma warning disable CS4014
-                Task.Run(async () =>
+                try
                 {
-                    using (var proxy = new NetworkStreamProxy(stream))
+                    NetworkStream stream =
+                        await _turnClient.AcceptRelayedStreamAsync();
+
+#pragma warning disable CS4014
+                    Task.Run(async () =>
```

Comment:
Even if we don't need the result, wouldn't it be good to `await` the task at the end of this `while` loop?

Reply:
If we `await` this task at here, we can't accept other relay stream same time because `await`ing will blocks this loop.

Reply:
@longfin Yes, of course we shouldn't `await` them *in the loop*.  So I suggested `await` them *at the end of this loop* so that these threads are joined to the current thread after the swarm stops.

Reply:
Indeed, apparently my suggestion cannot be applied without more structural change — as we can't maintain the unlimited numbers of `Task` objects (it depends on the number of requests, right?).  In the future, we should make an explicit thread pool and also take the number of workers to be pooled as a constructor parameter, I think.